templates/vue/src/components/TapestryDepthSlider.vue

```diff
@@ -97,7 +97,7 @@ export default {
   methods: {
     ...mapMutations(["updateNode"]),
     setDefaultDepth() {
-      this.currentDepth = this.settings.defaultDepth || this.maxDepth + 1
+      this.currentDepth = this.settings.defaultDepth || 3
```

Comment:
`3` should be stored as a constant in the app or at minimum as a data variable. It's not a good idea to hard-code values in the code.